File: ruby/vristo-ruby-starter/app/controllers/auth_controller.rb
class AuthController < ApplicationController
    def auth_boxed_signin
       render layout: "auth", :template =>'auth/boxed-signin'
    end

    def auth_boxed_signup
       render layout: "auth", :template =>'auth/boxed-signup'
    end

    def auth_boxed_lockscreen
       render layout: "auth", :template =>'auth/boxed-lockscreen'
    end

    def auth_boxed_password_reset
       render layout: "auth", :template =>'auth/boxed-password-reset'
    end

    def auth_cover_login
       render layout: "auth", :template =>'auth/cover-login'
    end

    def auth_cover_register
       render layout: "auth", :template =>'auth/cover-register'
    end

    def auth_cover_lockscreen
       render layout: "auth", :template =>'auth/cover-lockscreen'
    end

    def auth_cover_password_reset
       render layout: "auth", :template =>'auth/cover-password-reset'
    end
end

File: cakephp/vristo-cakephp-main/src/Controller/AuthController.php
<?php
declare(strict_types=1);

namespace App\Controller;
class AuthController extends AppController
{
    public function auth_boxed_signin()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/boxed-signin');
    }

    public function auth_boxed_signup()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/boxed-signup');
    }

    public function auth_boxed_lockscreen()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/boxed-lockscreen');
    }

    public function auth_boxed_password_reset()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/boxed-password-reset');
    }

    public function auth_cover_login()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/cover-login');
    }

    public function auth_cover_register()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/cover-register');
    }

    public function auth_cover_lockscreen()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/cover-lockscreen');
    }

    public function auth_cover_password_reset()
    {
        $this->viewBuilder()->setLayout('auth');
        return $this->render('/auth/cover-password-reset');
    }
}
